Slim/Formats/FLAC.pm

```diff
@@ -265,7 +265,7 @@ sub _addInfoTags {
 	$tags->{SIZE}       = $info->{file_size};
 	$tags->{SECS}       = $info->{song_length_ms} / 1000;
 	$tags->{OFFSET}     = 0; # the header is an important part of the file. don't skip it
-	$tags->{BITRATE}    = sprintf "%d", $info->{bitrate};
+	$tags->{BITRATE}    = sprintf "%d", ($info->{bitrate} || $info->{bitrage_ogg});
```

Comment:
Is there a bit of a rage in Ogg? 😁 

Don't know whether this typo is needed for compatibility with Audio::Scan?